scripts/python-version-check.py

```diff
@@ -0,0 +1,14 @@
+from sys import argv, exit, version_info
```

Comment:
Ah three minor nits:
1. rename to python-version-check ?
2. add interpreter / just so default python3 will be executed if not directly invoked by the /path/to/python python-version-check ?
  #!/usr/bin/env python3
3. License header ?

Reply:
I normally advocate for dropping the `.py`, but we intend to use this with the python version determined in the makefile (which could be overridden).

Do we have a guide on how to do the license header correctly?

Reply:
Ok, then we can leave .py. Yes after we discussed this offline I've added one here:

https://github.com/konveyor/pelorus/blob/master/scripts/install_dev_tools